devito/types/sparse.py

```diff
@@ -488,7 +488,10 @@ def coordinates(self):
 
     @property
     def coordinates_data(self):
-        return self.coordinates.data.view(np.ndarray)
+        try:
+            return self.coordinates.data.view(np.ndarray)
```

Comment:
`.data._local`

Reply:
since it gets fixed in your PR, I'm merging